src/Update.hs

```diff
@@ -144,13 +144,9 @@ updateLoop o log (Right (pName, oldVer, newVer, url) : moreUpdates) mergeBaseOut
       updateLoop o log moreUpdates mergeBaseOutpathsContext
 
 -- Arguments this function should have to make it testable:
-
--- * the merge base commit (should be updated externally to this function)
-
--- * the merge base context should be updated externally to this function
-
--- * the commit for branches: master, staging, staging-next, python-unstable
-
+-- - the merge base commit (should be updated externally to this function)
+-- - the merge base context should be updated externally to this function
+-- - the commit for branches: master, staging, staging-next, python-unstable
```

Comment:
Ormolu *hates* whitespace and tries really hard to exterminate it. The comment guard blocks are just to try to pad this very large function into some sections for human readability.